### PR TITLE
PHC-5429 - Add PillarsTile to TileList

### PR DESCRIPTION
**Pull request by namick**
## Changes

  -  Add PillarsTile to TileList

## Screenshots

Coming in next PR

### CODE REVIEW

File: src/components/tiles/TilesList.tsx [hunk 1]
```diff
@@ -38,6 +41,22 @@ export function TilesList({ styles: instanceStyles }: Props) {
 
   return (
     <ScrollView testID={tID('tiles-list')} style={styles.scrollView}>
+      {pillarsTileEnabled && (
+        <PillarsTile
+          onOpenDetails={(tracker, valuesContext) => {
+            const screenName =
+              pillarSettings?.advancedScreenTrackers?.includes(
+                tracker.metricId || tracker.id,
+              ) || true
```
Comment (sternetj) [inline src/components/tiles/TilesList.tsx:50]: I assume this was for debugging.
```suggestion
              )
```

File: src/navigators/HomeStack.tsx [hunk 2]
```diff
@@ -38,6 +60,37 @@ export function HomeStack() {
...
+      <Item
+        title="Save"
+        onPress={async () => {
+          await new Promise(notifySaveEditTrackerValue);
+          navigation.goBack();
+        }}
+      />
+    </HeaderButtons>
+  );
+};
```
Comment (sternetj) [inline src/navigators/HomeStack.tsx:96]: Might be nice to move this to it's own file.

**Review by sternetj — APPROVED** [review]

**Review by markdlv — APPROVED** [review]
+1 on @sternetj's suggestions

File: src/components/tiles/TilesList.tsx (hunk 1 above)
Comment (namick) [reply src/components/tiles/TilesList.tsx:50]: Yes, removed.

**Pull request merged**